code/datums/martial/boxing.dm

```diff
@@ -100,3 +279,19 @@
 /obj/item/clothing/gloves/boxing/dropped(mob/user)
 	. = ..()
 	style.fully_remove(user)
+
+// Evil Boxing; for sick, evil scoundrels. 
+
+/datum/martial_art/boxing/evil
+	name = "Evil Boxing"
+	id = MARTIALART_EVIL_BOXING
+	pacifist_style = FALSE
+	honorable_boxer = FALSE
+
+/obj/item/clothing/gloves/boxing/evil
+	style_to_add =/datum/martial_art/boxing/evil
```

Comment:
Why not have the name and such here instead of defining them in two different places?

Reply:
I've moved all this, both the normal and evil gloves to the boxing glove file instead.